### ISSUE TITLE
Consider uBar 

### ISSUE THREAD

**Issue by mhenry** [eczarny/spectacle#804]

**Comment by Nowaker** [comment]
Indeed. It would be great if going "Fullscreen" wouldn't go too full screen. ;-) Here's a visualization of what we mean:

Expected:
![image](https://user-images.githubusercontent.com/611746/32694383-11601754-c704-11e7-847c-93951fd95638.png)

Actual:
![image](https://user-images.githubusercontent.com/611746/32694380-08054012-c704-11e7-9b8d-5baa439ead6d.png)

**Comment by drewswaycool** [comment]
This issue seems to be more of a problem when using ubar across multiple monitors.

**Comment by int-a** [comment]
Has there been any fix or workaround for this? Just got uBar and am having the same issue.

**Comment by marketica** [comment]
I'm looking for a fix to this as well, anyone found a solution and/or alternative that works?
